Dalamud/Game/Text/SeStringHandling/SeString.cs

```diff
@@ -254,10 +254,10 @@ public static SeString CreateItemLink(Item item, bool isHq, string? displayNameO
     /// <param name="rawX">The raw x-coordinate for this link.</param>
     /// <param name="rawY">The raw y-coordinate for this link..</param>
     /// <returns>An SeString containing all of the payloads necessary to display a map link in the chat log.</returns>
-    public static SeString CreateMapLink(uint territoryId, uint mapId, int rawX, int rawY)
+    public static SeString CreateMapLink(uint territoryId, uint mapId, int rawX, int rawY, int? instance = null)
```

Comment:
This is a breaking API change. Please remove the default param and instead create an overload without the optional param:

```csharp
public static SeString CreateMapLink(uint territoryId, uint mapId, int rawX, int rawY) => 
    CreateMapLink(territoryId, mapId, rawX, rawY, null);

public static SeString CreateMapLink(uint territoryId, uint mapId, int rawX, int rawY, int? instance) {
   ...
```

Applies to all changed public methods.

Reply:
ahh yes will do! just curious (because i don't know enough about c# yet) is this breaking because when compiled, old plugins will still be trying to find the function without the extra param, despite the default?

Reply:
Yep, that’s the easy answer. We had a problem specifically related to this crop up today with an earlier PR already.

Reply:
oh actually, since we're doing overloads now instead of defaults, would it be better to place instance after `mapId` and `placeName` respectively, so it aligns with where it would appear in the link text?

Reply:
i did rearrange the function arguments, but if you'd prefer i add `instance` on the end i don't mind putting it back